Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.9-buster
+
+# Apt-get update and initial installations
+RUN apt-get update && apt-get install 
+RUN apt-get install vim -y --no-install-recommends
+# RUN apt-get install nodejs npm -y 
+
+# Copy source files and install py packages
+RUN mkdir -p /opt/flydar
+RUN mkdir -p /opt/flydar/pip_cache
+COPY . /opt/flydar
```

Comment:
This adds the current sqlite file in your code repo into the docker container, together with any existing records. Instead, we should probably use a docker volume (via a docker-compose.yml). In the past I've created a postgres DB, and done the setup as per here https://github.com/michaellisitsa/expense_tracker/blob/0354eed5463653777f5731008043bd3152128f55/project/settings.py#L134-L144
and added the environment variables in the `docker-compose.yml` file:
https://github.com/michaellisitsa/expense_tracker/blob/0354eed5463653777f5731008043bd3152128f55/docker-compose.yml#L8-L11

It looks like I've probably followed the official docker django docs to get the above here https://docs.docker.com/samples/django/